code/ui/.storybook/main.ts

```diff
@@ -17,7 +17,7 @@ const allStories = [
     titlePrefix: '@storybook-blocks',
   },
 ];
-const blocksOnlyStories = ['../blocks/src/**/*.stories.@(js|jsx|ts|tsx|mdx)'];
+const blocksOnlyStories = ['../blocks/src/@(blocks|controls)/**/*.@(mdx|stories.@(tsx|ts|jsx|js))'];
```

Comment:
only expose `./blocks` and `./controls` in the Blocks SB, as we only actually export from those directories. (expect for `PureArgsTable` in `./components` though)